### ISSUE TITLE
Licensing?

### ISSUE THREAD

**Issue by CelticMinstrel** [animetrics/PlistCpp#18]
There seems to be no license file in the repo, which technically means that no-one is allowed to use this in anything without explicit permission (presumably not your intent). Could it be made available under something non-GPL?

**Comment by animetrics** [comment]
Licensing is provided in the comments in the top of each source code file.  It is distributed under the MIT license.

**Comment by CelticMinstrel** [comment]
Ah, thanks. It'd be more convenient if this was mentioned in a standalone file though. (Plus, if done right, GitHub will show the license in the stats line along with commit count etc.)

**Comment by animetrics** [comment]
Fair enough, though putting it in the source avoids ambiguity regarding whether or not the library is covered in whole or in part by the license.
